descriptors/ModuleDescriptor-template.json

```diff
@@ -0,0 +1,124 @@
+{
+  "id": "${artifactId}-${version}",
+  "name": "EBSCO KB Broker",
+  "provides": [
+    {
+      "id": "kb-ebsco-java",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/packages"
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/eholdings/packages"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/packages/{packageId}"
+        },
+        {
+          "methods": ["PUT"],
+          "pathPattern": "/eholdings/packages/{packageId}"
+        },
+        {
+          "methods": ["DELETE"],
+          "pathPattern": "/eholdings/packages/{packageId}"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/packages/{packageId}/resources"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/providers"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/providers/{providerId}"
+        },
+        {
+          "methods": ["PUT"],
+          "pathPattern": "/eholdings/providers/{providerId}"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/providers/{providerId}/packages"
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/eholdings/resources"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/resources/{resourceId}"
+        },
+        {
+          "methods": ["PUT"],
+          "pathPattern": "/eholdings/resources/{resourceId}"
+        },
+        {
+          "methods": ["DELETE"],
+          "pathPattern": "/eholdings/resources/{resourceId}"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/titles"
+        },
+        {
+          "methods": ["POST"],
+          "pathPattern": "/eholdings/titles"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/titles/{titleId}"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/proxy-types"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/root-proxy"
+        },
+        {
+          "methods": ["PUT"],
+          "pathPattern": "/eholdings/root-proxy"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/configuration"
+        },
+        {
+          "methods": ["PUT"],
+          "pathPattern": "/eholdings/configuration"
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/eholdings/status"
+        }
+      ]
+    },
+    {
+      "id": "_tenant",
+      "version": "1.0",
+      "interfaceType": "system",
+      "handlers": [
+        {
+          "methods": ["POST"],
+          "pathPattern": "/_/tenant"
+        }
+      ]
+    }
+  ],
+  "permissionSets" : [],
```

Comment:
We have been pretty relaxed with `permissionSets` and `requires` in mod-kb-ebsco but I think we should put them together here. Good example that we can refer to : https://github.com/folio-org/mod-codex-ekb/blob/master/descriptors/ModuleDescriptor-template.json
Its ok if we do not want to address it in this PR - but if thats the case, can we please create a user story in the backlog so it does not slip through cracks? Thanks!

Reply:
sure, @sduvvuri-ebsco. I will create a user story for adding permissions tomorrow.

Reply:
Thanks @natali-zaitseva!

Reply:
there is a MODKBEBSCO-14 for permissions. I'll discuss with Khalilah if it's the right one